webapp/components/Table/tableLink.js

```diff
@@ -0,0 +1,12 @@
+export const getLimit = () => 15
```

Comment:
should we store this limit as constant?

Reply:
nope. this will be needed. I will explain later why this will be dynamic